docs/internals/contributing/new-contributors.txt

```diff
@@ -1,85 +1,54 @@
+But here is how you can get started and get on track as a new contributor.
 
 .. admonition:: Basic tools and workflow
 
     If you are new to contributing to Django, the :doc:`/intro/contributing`
     tutorial will give you an introduction to the tools and the workflow.
 
-First steps
-===========
+Django is hosted on Github_. You can find bugs and feature requests
+on Trac_.
 
-Start with these steps to discover Django's development process.
+**Pick a subject area**
+  Pick something that you care about, and that you are familiar with,
+  or that you want to learn about. You don't already have to be an
+  expert on the area you want to work on; you become an expert
+  through your ongoing contributions to the codebase.
 
-* **Triage tickets**
+  There are subject areas like:
 
-  If an `unreviewed ticket`_ reports a bug, try and reproduce it. If you
-  can reproduce it and it seems valid, make a note that you confirmed the bug
-  and accept the ticket. Make sure the ticket is filed under the correct
-  component area. Consider writing a patch that adds a test for the bug's
-  behavior, even if you don't fix the bug itself. See more at
-  :ref:`how-can-i-help-with-triaging`
+  * **Developers**
+      Writing Django code is one major subject area of contribution.
+      You can write patches for bugs or features and make pull requests
+      to be reviewed.
 
-* **Look for tickets that are accepted and review patches to build familiarity
-  with the codebase and the process**
+      You can start from the contributing :doc:`code guide <writing-code/index>`
 
-  Mark the appropriate flags if a patch needs docs or tests. Look through the
-  changes a patch makes, and keep an eye out for syntax that is incompatible
-  with older but still supported versions of Python. :doc:`Run the tests
-  </internals/contributing/writing-code/unit-tests>` and make sure they pass.
-  Where possible and relevant, try them out on a database other than SQLite.
-  Leave comments and feedback!
-
-* **Keep old patches up to date**
-
-  Oftentimes the codebase will change between a patch being submitted and the
-  time it gets reviewed. Make sure it still applies cleanly and functions as
-  expected. Updating a patch is both useful and important! See more on
-  :doc:`writing-code/submitting-patches`.
+  * **Documenting**
+      Django has a lot of documentation and most of them could use some editing
+      from you. :doc:`Improve the documentation <writing-documentation>`.
 
-* **Write some documentation**
+  * **Localizing**
+      Most part of Django are internationalized, and needs more than translators
+      to increase the language reach. :doc:`Help translate Django <localizing>`.
 
-  Django's documentation is great but it can always be improved. Did you find
-  a typo? Do you think that something should be clarified? Go ahead and
-  suggest a documentation patch! See also the guide on
-  :doc:`writing-documentation`.
+**Start small**
```

Comment:
This would need a blank line above, but it looks like part of _Guidelines_ below 🤔